test/e2e/saucelabs.conf.js

```diff
@@ -59,6 +65,9 @@ const setupConfig = {
     fileAcceptor.bootstrap(done);
   },
   teardownAll: done => {
+    // Pause to allow SauceLabs to finish updating before Jenkins queries it for results
+    console.log('Wait for 30 seconds before Jenkins queries SauceLabs results...');
```

Comment:
still one console.log here. It's cleaner to use a logger.

Reply:
good call, will add now